src/appleseed/renderer/modeling/environmentedf/oslenvironmentedf.cpp

```diff
@@ -76,11 +151,17 @@ namespace
     {
       public:
         OSLEnvironmentEDF(
-            const char*             name,
-            const ParamArray&       params)
+            const char*         name,
+            const ParamArray&   params)
           : EnvironmentEDF(name, params)
+          , m_importance_map_size(0)
+          , m_probability_scale(0.0f)
         {
             m_inputs.declare("osl_background", InputFormatEntity, "");
+
+            // TODO:
```

Comment:
I'm not sure to understand this TODO.

Reply:
Just a hint to add any additional parameters if needed in the future. But I have removed it because it sounds confusing. It is actually clear without a comment.